kalite/updates/static/js/updates/bundle_modules/update_videos.js

```diff
@@ -396,7 +396,7 @@ function withNodes(nodeKey, callback, currentNode) {
 
 function setNodeClass(nodeKey, className) {
     withNodes(nodeKey, function(node) {
-        $(node.span).removeClass("unstarted partial complete").extraClasses(className);
+        // $(node.span).removeClass("unstarted partial complete").extraClasses(className);  no idea why run this here?
```

Comment:
this line never gets run successfully because it should be "$(node.span).removeClass("unstarted partial complete").extraClasses = className;"
but the line below this line ("node.extraClasses = className;") should get the job done right?